kolibri/plugins/management/assets/src/actions.js

```diff
@@ -106,7 +106,7 @@ function assignUserRole(user, kind) {
  */
 function createUser(store, stateUserData) {
   const userData = {
-    facility: store.pageState.facility_id,
+    facility: stateUserData.facility_id,
```

Comment:
@indirectlylit I know you cleared out this pattern in [this commit](https://github.com/learningequality/kolibri/commit/8aeb2031e523dbad821099006d04c61dda3dae65), but the store object in actions (see below) doesn't have pageState as an attribute. Per [vuex 1's documentation](https://github.com/vuejs/vuex/blob/1.0/docs/en/state.md), it's meant for mutation dispatches almost exclusively.